apps/opik-frontend/src/components/pages/CompareOptimizationsPage/CompareOptimizationsTrialsTable.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { ColumnPinningState, ColumnSort } from "@tanstack/react-table";
+import { ColumnDef } from "@tanstack/react-table";
+import { ROW_HEIGHT, OnChangeFn } from "@/types/shared";
+import { Experiment } from "@/types/datasets";
+import { Card } from "@/components/ui/card";
+import DataTable from "@/components/shared/DataTable/DataTable";
+import DataTableVirtualBody from "@/components/shared/DataTable/DataTableVirtualBody";
+import DataTableNoData from "@/components/shared/DataTableNoData/DataTableNoData";
+import { DataTableWrapperProps } from "@/components/shared/DataTable/DataTableWrapper";
+import { TABLE_WRAPPER_ATTRIBUTE } from "@/components/layout/PageBodyStickyTableWrapper/PageBodyStickyTableWrapper";
+
+export const getRowId = (e: Experiment) => e.id;
+
+export const DEFAULT_COLUMN_PINNING: ColumnPinningState = {
+  left: ["name"],
+  right: [],
+};
+
+const StickyTableWrapperWithBorder: React.FC<DataTableWrapperProps> = ({
+  children,
+}) => {
+  return (
```

Comment:
NIT: Think we should have, the eslint rule, in tech debt, for arrow value, to use => (), without braces and return 😄 

But maybe not this PR, just to ensure we have it as a jira ticket